### ISSUE TITLE
Floating clay, instead of baby

### ISSUE THREAD

**Issue by Nimrod108** [GenMode/Minecraft#14]
Baby animals are replaced by floating clay. All dependencies installed. Version 1.20.1, Fabric

**Comment by tionard** [comment]
same

**Comment by GenMode** [comment]
are you using the mod or the datapack version? (also very sorry that I didn't check this until now)

**Comment by tionard** [comment]
I've had mod version and the resource pack, currently uninstalled but can install again if some tests are needed

**Comment by GenMode** [comment]
are you also in 1.20.1 Fabric or are you on a different minecraft update, and if you're on a different minecraft update what update version of Better Babies are you on?

**Comment by Kazumoria** [comment]
Hi I'm also having the same problem. I've tested it with and without other mods and the same thing happens with both, I'm running Forge 1.21.1. One interesting thing to note is that in the 1.21.1 version of the mods the mobs show up as clay and that in 1.21.3 they show up as poisonous potatoes. when I switch versions they stay as clay or potatoes until I turn off and on the models with the /function command. The sheep also show up as iron horse armor regardless of version for whatever reason. It would be a nice addition to my modpack so hopefully this helps, thanks! 
![2024-11-15_20 26 21](https://github.com/user-attachments/assets/7f6595eb-807e-4e66-89e3-6731e6ca6359)
![2024-11-15_20 37 38](https://github.com/user-attachments/assets/1d566019-78c2-45ef-ba26-b40fd5037206)

**Comment by GenMode** [comment]
since you're using the mod do you have it installed client side as well? or only serverside?

**Comment by Kazumoria** [comment]
Yes I'm playing in single player with these screenshots, haven't actually tested it server side yet since I'm just building the mod base.

**Comment by GenMode** [comment]
what modloader are you using? fabric/forge/neoforge/quilt? and if you're using a launcher what launcher are you using too?

**Comment by Kazumoria** [comment]
Using forge 52.0.28 on the Minecraft launcher.

**Comment by GenMode** [comment]
if it is possible can you send me your modlist (not the server modlist but the one you're using on your client)?

**Comment by Kazumoria** [comment]
I'll attach a screenshot to all of the mods. I've tested it without mods tho so I'm not sure if it would change anything? I also tested the config folder by deleting everything just to make sure and that didn't help either. I'm not sure how you integrated the mods together but Is it possible that the data api mod and better babies have conflicting versions? I have data api 1.6.3 and better babies 0.7.13

![Modlist](https://github.com/user-attachments/assets/7b43ef14-5fc6-46cf-b025-6723c921ae59)

**Comment by GenMode** [comment]
thank you very much this was helpful. May I ask where you downloaded better babies from?

**Comment by Kazumoria** [comment]
I downloaded them from CurseForge, but I also tried the Modrinth downloads for them and still got the bug. And thanks for your help as well, without people like you we wouldn't have all these awesome mods! I'm currently working on a mod as well and I hope I'm able to become a full time developer, Minecraft modding feels like the best place to start C:

**Comment by GenMode** [comment]
thank you for the compliment and sorry for this taking long

Ok that being said I think I know the problem. See this is a datapack/mod not just a regular mod. This means that without crashing they can be added serverside but textures won't show up unless you have it installed client side too. Unfortunately it seems both websites aren't abundantly clear about that so I'll be updating the descriptions to better reflect that soon

TL;DR: please install the mod client side AS WELL as serverside since your client cant read the models without it, and when you do please update me so that I know if it worked or not. Sorry for the confusion

**Comment by Kazumoria** [comment]
Ahh okay sounds good, I'll get a server set up right now and see how that works, thanks for the help!

**Comment by Kazumoria** [comment]
Alright so I tried it out and somehow that didn't work. I tried disabling and re-enabling the entities with the /function command but that didn't do anything either. I'll attach a screenshot of my f3 confirming I'm on a server. I only had betterbabies and data api installed. I'll also attach my debug.log and normal log if those are of any help. The server was made using the forge 52.0.28 jar.  No worries about it taking long btw, I joined your discord and said hi in chat so if you wanted to message through there that would probably be faster, but either way it's no worries cause stuff like this just takes time.
[latest.log](https://github.com/user-attachments/files/17794350/latest.log)
[debug.log](https://github.com/user-attachments/files/17794353/debug.log)
![2024-11-17_19 23 25](https://github.com/user-attachments/assets/e94c0702-50b0-4f41-acc5-4d8d11c3ccc3)

**Comment by GenMode** [comment]
yes it might be faster there since I get ping sounds when I'm messaged
